File: training-data/java/RemoveDuplicates.java
import java.util.LinkedHashSet;
import java.util.List;

public class RemoveDuplicates {
    public static List<Integer> removeDuplicatesOrdered(List<Integer> items) {
        return new java.util.ArrayList<>(new LinkedHashSet<>(items));
    }

    public static void main(String[] args) {
        System.out.println(removeDuplicatesOrdered(List.of(1, 2, 2, 3, 1, 4)));
    }
}

File: training-data/csharp/RemoveDuplicates.cs
using System;
using System.Linq;

class RemoveDuplicates
{
    static int[] RemoveDuplicatesOrdered(int[] items) => items.Distinct().ToArray();

    static void Main()
    {
        Console.WriteLine(string.Join(" ", RemoveDuplicatesOrdered(new[] { 1, 2, 2, 3, 1, 4 })));
    }
}
